index.bs

```diff
@@ -5219,6 +5226,17 @@ type.
         "Promise" "&lt;" ReturnType "&gt;"
 </pre>
 
+<pre class="grammar" id="prod-OpenDictionaryType">
+    OpenDictionaryType :
+        "OpenDictionary" "&lt;" Type OpenDictionaryTypeMaybeValue
```

Comment:
Before switching this to require both arguments, I'd like a couple more people to agree with @tobie.